src/pages/ProfilePage/components/UserLikedPosts/UserLikedPosts.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from "react";
+import styles from "./Liked.module.scss";
+
+const Liked: FC = () => {
+  return (
+    <div className={styles["liked-wrapper"]}>
+      {[...new Array(10)].map((like, index) => (
```

Comment:
An error happens during building because "like" param is not used. To avoid this failure, you can write "_like" to signilize the linter that this param should not be used

Reply:
It's a placeholder for posts to see how it will be, it's not finished yet, I'll replace it with real post's latter. For now I remove this code.